docs/content/docs/deployment/elastic_scaling.md

```diff
@@ -0,0 +1,91 @@
+# Submit Job in Application Mode
+./bin/standalone-job.sh start -Drestart-strategy=fixeddelay -Drestart-strategy.fixed-delay.attempts=100000 -Djobmanager.scheduler=adaptive -Dscheduler-mode=reactive -j org.apache.flink.streaming.examples.windowing.TopSpeedWindowing
+# Start first TaskManager
+./bin/taskmanager.sh start
+```
+
+Let's quickly examine the used configuration parameters:
+- `-Drestart-strategy=fixeddelay` and `-Drestart-strategy.fixed-delay.attempts=100000` configure the job to restart on failure. This is needed for supporting scale-down.
+- `-Djobmanager.scheduler=adaptive` enables the Adaptive Scheduler, which is needed to use Reactive Mode.
+- `-Dscheduler-mode=reactive` enables Reactive Mode.
+
+You have now started a Flink job in Application Mode. The [web interface on localhost:8081](http://localhost:8081) now shows that the job is running on one TaskManager. If you want to scale up the job, add another TaskManager to the cluster:
+```bash
+# Start additional TaskManager
+./bin/taskmanager.sh start
+```
+
+To scale down, remove a TaskManager instance.
+```bash
+# Remove a TaskManager
+./bin/taskmanager.sh stop
+```
+
+### Limitations
+
+Since Reactive Mode is a new, experimental feature, not all features supported by the default scheduler are also available with Reactive Mode (and its adaptive scheduler). The Flink community is working on addressing these limitations.
+
+- **Deployment is only supported as a standalone application** deployment. Active resource managers (such as native Kubernetes, YARN or Mesos) are explicitly not supported. Standalone session clusters are not supported either. The application deployment is limited to single job applications.
+- **Streaming jobs only**: The first version of Reactive Mode runs with streaming jobs only. When submitting a batch job, then the default scheduler will be used.
+- **No support for local recovery**: Local recovery is a feature that schedules tasks to machines so that the state on that machine gets re-used if possible. The lack of this feature means that Reactive Mode will always need to restore the entire state from the checkpoint storage.
+- **No support for local failovers**: Local failover means that the scheduler is able to restart parts ("regions" in Flink's internals) of a failed job, instead of the entire job. This limitations impacts only recovery time of embarrassingly parallel jobs -- Flink's default scheduler can restart failed parts, while Reactive Mode will restart the entire job.
+- **Limited integration with Flink's Web UI**: Reactive Mode allows that a job's parallelism can change over its lifetime. The web UI only shows the current parallelism of a job, not the historic evolution of the job. There might be other inconveniences such as a lack of the exception history, incorrect job status time-stamps and incorrect metrics.
+- **No support for fine grained resource specifications**: Fine-grained resource specifications are ignored by Reactive Mode.
```

Comment:
Like to documentation on fine-grained resource specifications.

Reply:
I fear that there's no documentation available yet: https://issues.apache.org/jira/browse/FLINK-14187?focusedCommentId=17295371&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-17295371

Reply:
I'd suggest to remove the bullet for fine-grained resource specifications for the moment. This feature is unlikely to be ready for the users to use in the 1.13 release. Thus, we are not exposing public interfaces atm.

Reply:
Thanks a lot! I've remove it!